PyInstaller/hooks/hook-PyQt6.QtWebEngineQuick.py

```diff
@@ -0,0 +1,14 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2014-2022, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+from PyInstaller.utils.hooks.qt import add_qt5_dependencies
+
+hiddenimports, binaries, datas = add_qt5_dependencies(__file__)
```

Comment:
Strictly speaking, this should be `add_qt6_dependencies` (although at the moment, both map to the same function behind the scenes).